StreamChatClient_v3/Database/DTOs/ChannelDTO.swift

```diff
@@ -83,6 +82,19 @@ extension NSManagedObjectContext {
         return dto
     }
     
+    func saveChannel<ExtraData: ExtraDataTypes>(payload: ChannelPayload<ExtraData>,
+                                                query: ChannelListQuery?) throws -> ChannelDTO {
```

Comment:
It's not so obvious of difference between these 2 methods `saveChannel` with the same parameter names and no comments to methods. Can you add comments or change parameter names to make it more distinguish?

Reply:
And again this method call the previous method. I think the order should be opposite.

Reply:
This overload is intentional. The idea is, you have some kind of `Channel` payload and you just call `saveChannel` with it and it works.